ktor-client/ktor-client-core/common/src/io/ktor/client/features/cookies/CookiesStorage.kt

```diff
@@ -31,7 +31,9 @@ suspend fun CookiesStorage.addCookie(urlString: String, cookie: Cookie) {
 }
 
 internal fun Cookie.matches(requestUrl: Url): Boolean {
-    val domain = domain?.toLowerCasePreservingASCIIRules()?.trimStart('.') ?: error("Domain field should have the default value")
+    val domain = domain?.toLowerCasePreservingASCIIRules()?.trimStart('.')
+        ?: error("Domain field should have the default value")
```

Comment:
Could you please improve the message to explain what one should do? Currently, it is not clear what domain we are talking about and where it should be.

Reply:
no problem. btw it doesn't change